ui/common/css/component/_user-link.scss

```diff
@@ -47,6 +47,10 @@
   }
 }
 
+.player-link {
```

Comment:
bit weird to add a class there to change the link color and call it `player-link`.

If we want user links to be blue in the chat, then maybe we should say so in the chat css instead?

Reply:
Do we not also want player links to be blue in the forums? Or do you think it would be better to add to add separately to both. 

I only added player-link because the user-link css overrides @player and changes it from blue to grey.

Reply:
> Do we not also want player links to be blue in the forums

That's outside the scope of this PR, which is about adding powerTips. richText.ts doesn't even run in the forum, it's only for chats.

Reply:
I think it is fixed now.